### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by HadikHuszar**
## Overview

- Added Stepper and configured it to my specs.
- Added a Secondary button to each step item.
- Was able to make everything work from object, **except for secondary button object**

## Feedback Requested

Currently blocked on getting button actions to be coming from the object.
I suspect that, as usual, it is a syntax issue.  Guidance would be helpful at this point.  Thank you!

### CODE REVIEW

File: app/src/App/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from "react";
 
+import { ChakraProvider } from "@chakra-ui/react";
```
Comment (serpient) [inline app/src/App/index.js]: are we still using chakra provider?

File: app/src/Tasks/stepper.js [hunk 2]
```diff
@@ -46,34 +69,48 @@ export default function VerticalLinearStepper() {
...
-              <Box sx={{ mb: 2 }}>
+              <Typography>
+                {step.description}
+                <Button
+                  variant="outlined"
+                  size="small"
+                  sx={{ mt: 1, mr: 1, mb: 1, ml: 5 }}
+                  onClick={() => {
+                    alert("clicked");
+                  }}
```
Comment (serpient) [inline app/src/Tasks/stepper.js]: this certainly looks right syntax-wise. I'll have to pull it down to take a look locally

Comment (HadikHuszar) [reply app/src/Tasks/stepper.js]: Okay cool ... looking forward to working through this issue with you.

File: app/src/App/index.js (hunk 1 above)
Comment (HadikHuszar) [reply app/src/App/index.js]: No.  I removed it on the higher level file, but missed this one.  Thanks!

**Pull request merged**